.k8s/api.values.yml

```diff
@@ -101,7 +101,7 @@ deployment:
         - |
           retry=120; # 5s * (12 * 10) = 10min
           while
-            ! curl -sS --fail "${NLP_URL}" &&
+            ! true &&
```

Comment:
Is this init container still relevant ?

Reply:
I think so, I've just updated the command to the check if the nlp service is alive.